### PR TITLE
[Mod Update] Legalize voyager 1.0.1

### PR DESCRIPTION
**Pull request by tildejustin**
take two

source: https://github.com/tildejustin/voyager/tree/abb9738ac777536846fad917199ee7b485cc19d0
changes the implementation from a [dastardly overwrite](https://github.com/tildejustin/voyager/blob/7bbe8a5bb29176a3bdd916b06c3f3d65c58bbb9b/src/main/java/me/modmuss50/voyager/mixin/StructureManagerMixin.java#L27-L49) to a [localized inject](https://github.com/tildejustin/voyager/blob/3240c3ce9fb95378a5c6e99424a08cd9710bb44a/src/main/java/me/modmuss50/voyager/mixin/StructureManagerMixin.java#L40-L46). this change has been battle-tested in fabric api and was just never released stand-alone. i also added a mixin config so as to avoid modifying any bytecode when java 10 or less was used, as the mod is unnecessary in that case.

### CODE REVIEW

File: .gitignore
Comment (DuncanRuns) [inline .gitignore:1]: this probably shouldn't be here

**Review by DuncanRuns — CHANGES_REQUESTED** [review]
Please remove .gitignore

**Review by DuncanRuns — APPROVED** [review]
LGTM
Please wait for leaderboard moderator approval before merging

**Comment by tildejustin** [comment]
note to self: change the inject method to `<init>*`

**Comment by tildejustin** [comment]
commit: https://github.com/tildejustin/voyager/commit/78bcc4a494c018ff3170619932bbf7982ff4dc65
adds support for older versions

**Comment by DuncanRuns** [comment]
I am not a big fan of that jar name

**Comment by tildejustin** [comment]
new source: https://github.com/tildejustin/voyager/tree/v1.0.1

**Review by DuncanRuns — APPROVED** [review]

**Pull request merged**